addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -6,17 +6,13 @@
 class AccountMoveSend(models.TransientModel):
     _inherit = 'account.move.send'
 
-    l10n_it_edi_warning_message = fields.Html(compute='_compute_l10n_it_edi_xml_export')
-
+    l10n_it_edi_warning_message = fields.Json(compute='_compute_l10n_it_edi_xml_export')
     l10n_it_edi_enable_xml_export = fields.Boolean(compute='_compute_l10n_it_edi_xml_export')
     l10n_it_edi_readonly_xml_export = fields.Boolean(compute='_compute_l10n_it_edi_xml_export')
     l10n_it_edi_checkbox_xml_export = fields.Boolean('E-invoice XML',
         compute='_compute_l10n_it_edi_checkbox_xml_export',
         store=True,
-        readonly=False,
-        help="Create the e-invoice XML ready to be sent to the Italian Tax Agency.\n"
```

Comment:
The help was not useful?

Reply:
It is, I just moved it here to match what happens in the email warning.
https://github.com/odoo/odoo/pull/142596/files/6c711cff56adcea14f1a5f25f4a50e093739eee3#diff-a1829d76bc9c78bd0734aa0439dedf0d3af06d856dc71007dce0e061b9d54d08R22